src/send_kcidb.py

```diff
@@ -141,11 +141,11 @@ def _parse_checkout_node(self, origin, checkout_node):
             'git_repository_branch':
                 checkout_node['data']['kernel_revision']['branch'],
             'git_commit_tags':
-                checkout_node['data']['kernel_revision']['commit_tags'],
+                checkout_node['data']['kernel_revision'].get('commit_tags'),
             'git_commit_message':
-                checkout_node['data']['kernel_revision']['commit_message'],
+                checkout_node['data']['kernel_revision'].get('commit_message'),
             'git_repository_branch_tip':
-                checkout_node['data']['kernel_revision']['tip_of_branch'],
+                checkout_node['data']['kernel_revision'].get('tip_of_branch', False),
```

Comment:
We can keep it `None` instead of `False` if the flag is not set.

Reply:
Ok